src/Symfony/Component/Messenger/Middleware/TraceableMiddleware.php

```diff
@@ -21,14 +21,12 @@
  */
 class TraceableMiddleware implements MiddlewareInterface
 {
-    private $inner;
     private $stopwatch;
     private $busName;
     private $eventCategory;
 
-    public function __construct(MiddlewareInterface $inner, Stopwatch $stopwatch, string $busName = null, string $eventCategory = 'messenger.middleware')
+    public function __construct(Stopwatch $stopwatch, string $busName = null, string $eventCategory = 'messenger.middleware')
```

Comment:
```diff
- string $busName = null
+ string $busName
```

? It's mandatory in TraceableStack which makes sense now as we don't have the shared middleware issue anymore.

Reply:
works for me, updated, thanks.